rasa/nlu/classifiers/diet_classifier.py

```diff
@@ -289,11 +298,12 @@ def __init__(
 
         # transform numbers to labels
         self.index_label_id_mapping = index_label_id_mapping
-        self.index_tag_id_mapping = index_tag_id_mapping
+        self.index_tag_id_mappings = index_tag_id_mappings
 
         self.model = model
 
-        self.num_tags: Optional[int] = None  # number of entity tags
+        # number of entity tags for every CRF
+        self.crf_num_tags: Optional[Dict[Text, int]] = None
```

Comment:
should this be private?
```suggestion
        self._crf_num_tags: Optional[Dict[Text, int]] = None
```